policyengine_us/variables/gov/states/az/hhs/tanf/az_hhs_tanf_earned_income_care_expense_disregard.py

```diff
@@ -0,0 +1,30 @@
+from policyengine_us.model_api import *
+
+
+class az_hhs_tanf_earned_income_care_expense_disregard(Variable):
+    value_type = float
+    entity = SPMUnit
+    label = "Arizona Cash Assistance care expense earned income disregard"
+    definition_period = MONTH
+    defined_for = StateCode.AZ
+
+    def formula(spm_unit, period, parameters):
+        p = parameters(
+            period
+        ).gov.states.az.hhs.tanf.eligibility.income.earned.care_expenses
+        # Get the age of the child(children) and the disabled adult(s)
+        person = spm_unit.members
+        age = person("age", period) * MONTHS_IN_YEAR
+        # Get the childcare and disabled adult care expenses
+        care_expenses = spm_unit("childcare_expenses", period)
+        # Determine the total eligible disregard
+        # The eligibility reuquirements consider whether children or disabled adults
+        is_child = person("az_tanf_eligible_child", period)
+        disabled_adult = person("is_disabled", period)
```

Comment:
need to check they're not a child right? please add a test for disabled children getting the right amount